src/lib/Scenes/Map/Components/ShowCard.tsx

```diff
@@ -58,7 +58,11 @@ export class ShowCard extends Component<ShowCardProps, ShowCardState> {
     const equal = isEqual(previousIds, currentIds)
 
     if (!this.state.isSaving && !equal && this.list) {
-      setTimeout(() => this.list.scrollToOffset({ offset: 0, animated: true }), 500)
+      setTimeout(() => {
+        if (this.list) {
```

Comment:
Classic!

Reply:
hahah - yep

Reply:
lol yup, that's the JS version of `[weak self]` bug